test/runtests.jl

```diff
@@ -24,21 +24,27 @@ julia = Base.julia_cmd().exec[1]
 end
 
 @testset "build_executable" begin
-    builddir = mktempdir()
     jlfile = joinpath(@__DIR__, "..", "examples", "hello.jl")
-    snoopfile = open(joinpath(builddir, "snoop.jl"), "w") do io
-        write(io, open(read, jlfile))
-        println(io)
-        println(io, "using .Hello; Hello.julia_main(String[])")
+    basedir = mktempdir()
+    relativebuilddir = "build"
```

Comment:
I think it makes sense to separate the `snoopfile` from the `builddir` as you've done here though. thanks.